.github/workflows/reviewdog.yml

```
@@ -22,7 +22,7 @@ jobs:
           curl -sfL \
             https://github.com/reviewdog/reviewdog/raw/master/install.sh | \
               sh -s -- -b $HOME/bin
-          echo ::add-path::$HOME/bin
+          echo $HOME/bin >> $GITHUB_PATH
```

```suggestion
          echo "$HOME/bin" >> $GITHUB_PATH
```
The examples on GitHub always have quotes here. Not sure if it's needed.

---

It makes no difference, but I can change.